docs/Targets.md

```diff
@@ -693,3 +693,15 @@ the monitor command sequence below:
 (gdb) mon reset init
 (gdb) mon psoc6 reset_halt
 ```
+
+## NXP iMX-RT1060/1062 (RT1060-EVK)
+
+The NXP iMX-RT1060 is a Cortex-M7 with a DCP coprocessor for SHA256 acceleration.
+Example configuration for this target is provided in `./config/examples/imx-rt1060.config`.
+
+DCP support can be enabled by using PKA=1 in the configuration file.
```

Comment:
Once https://github.com/wolfSSL/wolfssl/pull/3373 is merged the sub-module should be updated so PKA=1 builds.